docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Rust Exercism Track
+
+Thank you so much for contributing! :tada:
+We welcome pull requests of all kinds.
+As one of the many tracks in Exercism, contributions here should observe Exercism standards like the [Code of Conduct](https://exercism.io/code-of-conduct).
+This document articulates maintainers expectations of contributors.
+It describes what is specific about working on the Rust track and how you can help.
+
+## Ways to Contribute
+
+- improving existing exercises
+- creating new exercises
+- internal tooling
+
+## Merging Philosophy
+
+A [pull request](https://docs.github.com/en/github/getting-started-with-github/github-glossary#pull-request) should address one logical change.
+This could be small or big.
+
+For example, [#1174](https://github.com/exercism/rust/pull/1174) fixed a single typo in a single file after minutes of collaboration.
```

Comment:
```suggestion
For example, [#1175](https://github.com/exercism/rust/pull/1175) fixed a single typo in a single file after minutes of collaboration.
```

Fairly sure that was just an off-by-one error.

Reply:
Perfect, thank you!